bin/bout-config.in

```diff
@@ -24,7 +24,6 @@ libs="@CONFIG_LDFLAGS@"
 
 version="@BOUT_VERSION@"
 git="@GIT_REVISION@"
-checks="@CHECK_LEVEL@"
```

Comment:
Is this still needed? I think it's printed further down the file

Reply:
No, its printed a bit further up - that's why I removed this duplicate.